app/views/RoomActionsView/ActionsList/index.js

```diff
@@ -0,0 +1,222 @@
+import React, { memo } from 'react';
+import { View, ScrollView } from 'react-native';
+import { isEqual } from 'lodash';
+import PropTypes from 'prop-types';
+
+import I18n from '../../../i18n';
+import RocketChat from '../../../lib/rocketchat';
+
+import ActionItem from './ActionItem';
+import RoomItem from './RoomItem';
+
+import scrollPersistTaps from '../../../utils/scrollPersistTaps';
+import styles from './styles';
+
+const areEqual = (prev, next) => isEqual(prev.room, next.room)

```

Comment:
I think you compare all of props, what you think about use only `isEqual(prev, next)`?